src/modules/sensors/vehicle_air_data/VehicleAirData.cpp

```diff
@@ -110,6 +110,17 @@ void VehicleAirData::SensorCorrectionsUpdate(bool force)
 	}
 }
 
+void VehicleAirData::AirTemperatureUpdate()
+{
+	differential_pressure_s differential_pressure;
+
+	// update air temperature if data from differential pressure sensor is in valid range
+	if (_differential_pressure_sub.update(&differential_pressure) && PX4_ISFINITE(differential_pressure.temperature)
+	    && fabsf(differential_pressure.temperature) < 50.f) {
```

Comment:
Why 50? It's not really valid, but on a hot day sitting in the sun preflight I've seen these exceed 50C (black wing, etc).